frontend/src/App.svelte

```diff
@@ -102,9 +119,17 @@ let activePage = $state("home");
     margin-right: 0.2rem;
     padding: 0.3rem;
   }
+  .navbuttons {
+    vertical-align: middle;
```

Comment:
todo: look at this

Reply:
i was experimenting with stuff, this has no effect now